packages/gafl-webapp-service/src/pages/terms-and-conditions/__tests__/route.spec.js

```diff
@@ -1,16 +1,74 @@
 import { getData, validator } from '../route'
 import pageRoute from '../../../routes/page-route.js'
 import { nextPage } from '../../../routes/next-page.js'
+import { LICENCE_SUMMARY, CONTACT_SUMMARY } from '../../../uri.js'
 
-jest.mock('../../../routes/next-page.js', () => ({
-  nextPage: jest.fn()
-}))
 jest.mock('../../../routes/page-route.js')
 
+const getMockPermission = price => ({
+  licensee: {
+    firstName: 'Turanga',
+    lastName: 'Leela'
+  },
+  licenceType: 'trout-and-coarse',
+  numberOfRods: '2',
+  licenceLength: '8D',
+  permit: { cost: price }
+})
+
 describe('terms-and-conditions > route', () => {
   describe('default', () => {
     it('should call the pageRoute with terms-and-conditions, /buy/conditions, validator, nextPage and getData', async () => {
       expect(pageRoute).toBeCalledWith('terms-and-conditions', '/buy/conditions', validator, nextPage, getData)
     })
   })
+
+  describe('getData', () => {
+    const generateMockRequest = (statusGet = async () => {}, transactionGet = async () => {}) => ({
+      cache: () => ({
+        helpers: {
+          transaction: {
+            get: transactionGet
+          },
+          status: {
+            getCurrentPermission: statusGet
+          }
+        }
+      })
+    })
+
+    beforeEach(() => jest.clearAllMocks())
+
+    it('LICENCE_SUMMARY redirect', async () => {
+      const status = () => ({
+        [LICENCE_SUMMARY.page]: false
+      })
```

Comment:
This (and other status mocks) should be async.
However...
You don't actually need to mock the status getter, just the object that it returns. So change `generateMockRequest` to take statusCache and transactionCache, something like:
```
    const generateMockRequest = (statusCache = {}, transactionCache = {}) => ({
       cache: () => ({
         helpers: {
           transaction: {
             get: async () => transactionCache
           },
           status: {
             getCurrentPermission: async () => statusCache
           }
         }
       })
     })
```
Then here you can just pass the object `{ [LICENCE_SUMMARY.page]: false }` to `generateMockRequest`